src/test/java/org/mifos/connector/airtel/AirtelConnectorApplicationTests.java

```diff
@@ -0,0 +1,12 @@
+package org.mifos.connector.airtel;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class AirtelConnectorApplicationTests {
+
+    @Test
```

Comment:
Can remove this as we dont add junit tests

Reply:
Done.